v1/lib/server/utils.js

```diff
@@ -66,30 +66,10 @@ function autoPrefixCss(cssContent) {
     .then(result => result.css);
 }
 
-// Validate the docs in the sidebar are valid
-function validateSidebar(metadata, sidebarMetadatas) {
```

Comment:
This function is too coupled to the implementation of the sidebars and since I changed the implementation it's no longer that relevant (my implementation doesn't generate `sidebarMetadatas`). It's a good idea to do validation but I don't think it's very necessary.